### PR TITLE
Support for Goerli

### PR DESCRIPTION
**Pull request by cristidas**

### CODE REVIEW

File: config/goerli.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
+{
+  "rpcURL": "https://kovan.infura.io/v3/11465e3f27b247eb8b785c23047b29fd",
```
Comment (jeannettemcd) [inline config/goerli.json:2]: this is for Goerli is it not ? You will need and rpc endpoint to goerli netowork

File: src/startGoerli.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import Keeper from './keeper.js';
+import path from 'path';
+
+const configPath = path.join(path.resolve(), 'config/goerli.json');
+const passwordPath = path.join(path.resolve(), '/wallet/jsonpassword.txt');
+const keystorePath = path.join(path.resolve(), '/wallet/testwallet_goerli.json');
```
Comment (jeannettemcd) [inline src/startGoerli.js:6]: Does this file need to exist?

**Comment by dharmendrakariya** [comment]
Is this PR valid? coz I think that `MakerOTCSupportMethods` , `OasisDex`, and `Callee` addresses (oasisCallee and uniswapCallee) ! are not there!

**Comment by cristidas** [comment]
Not needed anymore

**Pull request closed without merging**